lua/neotest-vitest/init.lua

```diff
@@ -37,9 +35,11 @@ end
 
 ---@return boolean
 local function hasRootProjectVitestDependency()
+  local rootPackageJson = vim.loop.cwd() .. "/package.json"
+
   local success, packageJsonContent = pcall(lib.files.read, rootPackageJson)
   if not success then
-    print("cannot read package.json")
+    print("cannot read package.json, got " .. rootPackageJson)
```

Comment:
It became MUCH easier to find the problem when I modified this print. I think it might be useful for plugin users as well! They will clearly see what's wrong with the calculated path.

Though we might want to modify message and print something more descriptive than just 'got'